play/PlayerRules/CharacterCreation/Class-Specific Documentation/Power Suits/Chassis.txt

```diff
@@ -0,0 +1,19 @@
+		Light Chassis
+Chassis Name:	Mantis
+Chassis Power Req:	3 PU
+Standard Speed:		1m/action
+Maneuver Bonus:		2m/action
+
+Fort Bonus:		2
+Reflex Penalty:		30
+Perception Penalty:	25
+
+Disrupt DC:		75
+Pwn DC:			120
+
```

Comment:
We could use fort and specify -does not grant additional death saving throws-, or go for the reverse and specify each piece exactly: +30  health, +6% Fort Save, +6% etc.

That implies a fairly low-tech suit then? But i'm with you on the concept.

👍